eve/main.yml

```diff
@@ -2522,29 +2522,6 @@ stages:
             <<: *_env_bastion_tests
             PYTEST_FILTERS: "solution"
       - SetPropertyFromCommand: *set_bootstrap_cp_ip_ssh
```

Comment:
Sorry @ChengYanJin
This one also
(It's the last one :smile: )

Reply:
Thanks, @TeddyAndrieux !!!
Would be great if you can 3rd time check if it looks good to you 😆 
https://github.com/scality/metalk8s/pull/3167/commits/640cee8b92d69a9d7efbdffb7419bf45b243c473